src/particles/plasma/PlasmaParticleContainer.cpp

```diff
@@ -446,6 +452,210 @@ IonizationModule (const int lev,
     }
 }
 
+void
+PlasmaParticleContainer::
+LaserIonization (const int islice,
+                 const amrex::Geometry& laser_geom,
+                 const MultiLaser& laser,
+                 const amrex::Real background_density_SI)
+{
+    if (!m_can_laser_ionize || !laser.UseLaser(islice)) return;
+    HIPACE_PROFILE("PlasmaParticleContainer::LaserIonization()");
+
+    using namespace amrex::literals;
+    using Complex = amrex::GpuComplex<amrex::Real>;
+    constexpr Complex I(0.,1.);
+
+    const PhysConst phys_const = get_phys_const();
+
+    // Loop over particle boxes with both ion and electron Particle Containers at the same time
+    for (amrex::MFIter mfi_ion = MakeMFIter(0, DfltMfi); mfi_ion.isValid(); ++mfi_ion)
+    {
+        // Extract laser array
+        Array3<const amrex::Real> const laser_arr = laser.getSlices().const_array(mfi_ion);
+
+        const CheckDomainBounds laser_bounds {laser_geom};
+
+        // Extract properties associated with physical size of the box
+        const amrex::Real dx_inv = laser_geom.InvCellSize(0);
+        const amrex::Real dy_inv = laser_geom.InvCellSize(1);
+        const amrex::Real dzeta_inv = laser_geom.InvCellSize(2);
+
+        // Offset for converting positions to indexes
+        amrex::Real const x_pos_offset = GetPosOffset(0, laser_geom, laser_geom.Domain());
+        const amrex::Real y_pos_offset = GetPosOffset(1, laser_geom, laser_geom.Domain());
```

Comment:
Same behavior but better style consistency.
```suggestion
        amrex::Real const x_pos_offset = GetPosOffset(0, laser_geom, laser_geom.Domain());
        amrex::Real const y_pos_offset = GetPosOffset(1, laser_geom, laser_geom.Domain());
```